src/bench/readblock.cpp

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2022 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <bench/bench.h>
+#include <bench/data.h>
+
+#include <chainparams.h>
+#include <consensus/validation.h>
+#include <node/blockstorage.h>
+#include <streams.h>
+#include <util/system.h>
+#include <validation.h>
+
+using node::BLOCK_SERIALIZATION_HEADER_SIZE;
+using node::OpenBlockFile;
+using node::ReadBlockFromDisk;
+using node::ReadRawBlockFromDisk;
+
+static void SetupReadBlockTest()
+{
+    // Create params and set blocksdir to temporary folder (/tmp usually on linux)
+    SelectParams(CBaseChainParams::MAIN);
+    gArgs.ForceSetArg("-blocksdir", fs::temp_directory_path().string());
```

Comment:
Why can't you use the test setup m_path_root? Using a hardcoded constant is going to cause crashes if the test is run in parallel or the dir is dirty otherwise.